.gitignore

```diff
@@ -27,6 +27,7 @@ C:\\nppdf32Log\\debuglog.txt
 .deploy
 .s3cfg
 .tool-versions
+doc/licenses/temp_licenses.xml
```

Comment:
maybe use `tmp/something` for this? Or maybe even better Ruby's `Tempfile` for the temporary report.